ui/src/Components/SearchCandidates.jsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { Table, Button, Col, Form, FormGroup, Label, Input } from 'reactstrap';
+import { SearchBox } from './search-box/search-box.component';
+import NavBar from './NavBar';
+
+export default class SearchCandidates extends React.Component {
+	constructor(props){
+		super(props);
+
+		this.state = {
+			loading: true,
+			search: null,
+			props: props,
+			searchName: null,
+			searchEmail: null,
+			searchNumber: null,
+		};
+	}
+
+
+	// async componentDidMount()
+	// {
+	// 	const url = `https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/search`;
+	// 	const resp = await fetch(url);
+	// 	const data = await resp.json();
+	// 	this.setState({search: data, loading:false})
+	// }
+	componentDidMount() {
+		fetch('https://bixe448nsa.execute-api.us-west-1.amazonaws.com/dev/search')
+		  .then(response => response.json())
+		  .then(data => this.setState({ search: data, loading: false }));
+	}
+
+	/** Event Handlers for search fields
+		// TODO: Uncomment so state is updated, triggering a rerender of results
```

Comment:
Is this a to-do for future sprints?

Reply:
yes, just a starting point for the filter function!